src/main/kotlin/graphics/scenery/volumes/SceneryContext.kt

```diff
@@ -106,6 +106,22 @@ open class SceneryContext(val node: VolumeManager, val useCompute: Boolean = fal
             modified = true
         }
 
+        override fun setUniform1iv(name: String?, count: Int, value: IntArray?) {
+            TODO("Not yet implemented")
```

Comment:
These TODOs should be resolved, otherwise SceneryContext will bail if they're used in the future.